src/gui/mainwindow.cpp

```diff
@@ -1395,9 +1489,15 @@ void MainWindow::updateGUI()
         QString html = tr("DL speed: %1", "e.g: Download speed: 10 KiB/s").arg(Utils::Misc::friendlyUnit(status.payloadDownloadRate, true));
         html += "\n";
         html += tr("UP speed: %1", "e.g: Upload speed: 10 KiB/s").arg(Utils::Misc::friendlyUnit(status.payloadUploadRate, true));
-#endif
+#endif // Q_OS_UNIX
         m_systrayIcon->setToolTip(html); // tray icon
     }
+#else
+    if (status.payloadDownloadRate > 0)
+        QtMac::setBadgeLabelText(tr("%1/s").arg(Utils::Misc::friendlyUnit(status.payloadDownloadRate)));
```

Comment:
>Are you sure this is a correct suggestion?

After reading your explanation, obviously not.
IMO, it would be better to add this:
```c++
QtMac::setBadgeLabelText(tr("%1/s", "s is shorthand for seconds")
    .arg(Utils::Misc::friendlyUnit(status.payloadDownloadRate)));
```
and split the line please.